Remove specific emails from snapshot email

adammhaile opened the pull request:
The snapshot email was only ever sending to me, @smunilla, and @jupierce - none of us are involved anymore.

thiagoalessio commented:
/lgtm

tbielawa commented:
Close. Should be more like the other emails we changed lately though.

https://github.com/openshift/aos-cd-jobs/blob/master/jobs/build/release/Jenkinsfile#L130-L131

adammhaile commented:
@tbielawa <tbielawa@redhat.com> That's fine with me as long as it gets
updated. I mostly wanted to bring attention to it.

On Mon, Jul 29, 2019 at 8:31 AM Tim Bielawa <notifications@github.com>
wrote:

> Close. Should be more like the other emails we changed lately though.
>
>
> https://github.com/openshift/aos-cd-jobs/blob/master/jobs/build/release/Jenkinsfile#L130-L131
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/aos-cd-jobs/pull/1904?email_source=notifications&email_token=AAHJV2RJX45GBGBBQXET4QTQB3PIVA5CNFSM4IHRFS52YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3ARRBQ#issuecomment-515971206>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHJV2UHLZZRJTUVVGQTFUTQB3PIVANCNFSM4IHRFS5Q>
> .
>
